.generator/templates/api_client.mustache

```diff
@@ -70,6 +70,8 @@ class ApiClient(object):
         self.default_headers = {}
         if header_name is not None:
             self.default_headers[header_name] = header_value
+        if self.configuration.compress:
+            self.default_headers["Accept-Encoding"] = "gzip"
```

Comment:
```suggestion
            self.default_headers.setdefault("Accept-Encoding", "gzip")
```

Reply:
@jirikuncar do we allow other formats of compression besides `gzip`? if not, I don't see why we would need to check the header value if `compress` is set to True.

Reply:
forward compatibility?